CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 ## [Unreleased]
 
 ## [2.2.0] - 2018-02-22
+### Fixed
+- Issue that prevented older OS such as CentOS 5 from installing the Sensu RPM package as they are unsigned (@smbambling)
```

Comment:
Ah sorry, I should have been more explicit, could you move these and the two under `Added` to the `## [Unreleased]` section above? Once I cut the release, I rename `Unlreased` to the next version number.

Reply:
Nope thats my bad, I read this wrong and thought 2.2.0 wasn't cut yet. Fix this now.